samples/subsys/usb/uvc/README.rst

```diff
@@ -0,0 +1,183 @@
+.. zephyr:code-sample:: uvc
+   :name: USB Video webcam
+   :relevant-api: usbd_api video_interface
```

Comment:
No need to fix now (I mean it!) but in case there is another round of updates, just realized we now have a proper doxygen group for the UVC device api

```suggestion
   :relevant-api: usbd_api usbd_uvc video_interface
```

Reply:
Good catch! I will aggregate all the non-blocking fixes to this branch then...
https://github.com/tinyvision-ai-inc/zephyr/pull/new/pr_fix_uvc